src/System Application/App/Cryptography Management/src/DotNetReadOnlySpanWrapper/X509CertificateWrapper.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Security.Cryptography.X509Certificates;
+using System.Runtime.Versioning;
+
+namespace X509CertificateWrapper
```

Comment:
We no longer need this, right?

Reply:
Correct

Reply:
If you could get rid of the stuff no longer needed, we could move forward with this PR 😊

Reply:
Done :)